templates/project/res/xml/cdv_core_file_provider_paths.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+       Licensed to the Apache Software Foundation (ASF) under one
+       or more contributor license agreements.  See the NOTICE file
+       distributed with this work for additional information
+       regarding copyright ownership.  The ASF licenses this file
+       to you under the Apache License, Version 2.0 (the
+       "License"); you may not use this file except in compliance
+       with the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+       Unless required by applicable law or agreed to in writing,
+       software distributed under the License is distributed on an
+       "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+       KIND, either express or implied.  See the License for the
+       specific language governing permissions and limitations
+       under the License.
+-->
+<!-- 
+  Note: This File provider may only be used by the Cordova core
+  itself and should not be used for responding to Intents because
+  it will expose the app's private data folders.
+
+  For more information about FileProviders see:
+  https://developer.android.com/reference/androidx/core/content/FileProvider
+-->
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <!-- cordova.file.dataDirectory -->
+    <files-path name="files" path="." />
+    <!-- cordova.file.cacheDirectory -->
+    <cache-path name="cache" path="." />
+    <!-- cordova.file.externalDataDirectory -->
+    <external-files-path name="external-files" path="." />
+    <!-- cordova.file.externalCacheDirectory -->
+    <external-cache-path name="external-cache" path="." />
+    <!-- cordova.file.externalRootDirectory -->
+    <external-path name="external" path="." />
```

Comment:
Are we using all of these, or do we only need access to the cache-path?

Ideally, we only set up providers for the access we actually need, to reduce the potential for security concerns, but it's not entirely clear to me from the code how exactly these are being used. Maybe we do need all of them?

Reply:
I tested, and we only need `cache-path`.  Reading through https://developer.android.com/reference/androidx/core/content/FileProvider this makes sense.  We are only calling `context.getCacheDir()` so that should be the only one that we really need.